### ISSUE TITLE
Upload/send image to API

### ISSUE THREAD

**Issue by BluDay** [FaridSafi/react-native-gifted-chat#496]
I'm deeply sorry to address this as an issue as it obviously isn't...

It's only a simple question that I have, that I sadly couldn't find the answer to!

I'm currently developing this dating app for someone and I have to be done in a few days. I came across your package and thought it would be a good idea to implement this brilliant chat interface in the app.

I have this basic PHP API that can register users and authenticate them (the chat is also working), but all I'm wondering is, how can I upload a selected or taken image to my localhost server (via the API)? How do I get a custom action button on the composer field? Please don't judge me but I didn't think the examples were helpful enough. (Maybe it's just me haha)

Much respect for providing people this awesome interface, you're a hero!

Thx :)

**Comment by hadrienbbt** [comment]
Hello,
Maybe this would do the work ?
https://github.com/wkh237/react-native-fetch-blob
You can use RCTCameraRoll as well as in the awesome demo to pick photo(s).

**Comment by BluDay** [comment]
@hadrienbbt Thanks for the suggestion and support! I've already figured out how to upload an image without react-native-fetch-blob. I used the regular fetch function and passed a FormData object instead of a stringified JSON object. It allowed me to fetch the file in my .php script using $_FILES request method variable. If anyone still don't understand how to create a chat with GiftedChat, I'm willing to help out! Just reach out to me...
